packages/workflow/src/RoutingNode.ts

```diff
@@ -233,23 +232,9 @@ export class RoutingNode {
 					throw error;
 				}
 
-				interface AxiosError extends NodeError {
-					isAxiosError: boolean;
-					description: string | undefined;
-					response?: { status: number };
-				}
-
-				const routingError = error as AxiosError;
-
 				throw new NodeApiError(this.node, error as JsonObject, {
 					runIndex,
 					itemIndex: i,
-					message: routingError?.message,
-					description: routingError?.description,
-					httpCode:
-						routingError.isAxiosError && routingError.response
-							? String(routingError.response?.status)
-							: 'none',
```

Comment:
What I understand so far is:
* if the error has a level of `error` we report to sentry, otherwise no
* the NodeApiError` eventually extends `ApplicationError`, which sets the level to `error` by default
* if the error that is passed to `NodeApiError` is an axios error and the status code does not start with a 5 we set the level to `warning`:

https://github.com/n8n-io/n8n/blob/b6590d841dc4f53a10ccc42c60e6da6d15af012e/packages/workflow/src/errors/node-api.error.ts#L193-L195

Reply:
> how this change achieves this error not being reported to sentry

I should've clarified - this is cleanup from investigating the Sentry issue, to remove a bogus type assertion (i.e. asserting `AxiosError` but later checking anyway in the ternary) and to remove props that are being set in the constructor anyway.

The root cause is we're reporting 5xx errors from third parties to Sentry. I [asked about that](https://n8nio.slack.com/archives/C069HS026UF/p1714980657366549) weeks ago but got no answer, so not touching that yet.